src/Message/BitbucketImportCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Message;
+
+class BitbucketImportCommand

```

Comment:
```suggestion
class BitbucketImportCommand extends ImportCommand
```

You of couse need to create new abstract class `ImportCommand` https://www.php.net/manual/en/language.oop5.abstract.php

Reply:
Founded solution